wemake_python_styleguide/visitors/ast/complexity/overuses.py

```diff
@@ -49,6 +49,8 @@ class StringOveruseVisitor(base.BaseNodeVisitor):
         '\r\n',
         '\t',
         '|',
+        '"',
```

Comment:
What about `'`?

Reply:
Yeah, it's also a good candidate! @whysage could you please add it to the list?

Reply:
done